src/test/java/gov/healthit/chpl/aqa/pageObjects/ChplAPIPage.java

```diff
@@ -0,0 +1,46 @@
+package gov.healthit.chpl.aqa.pageObjects;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+/**
+ * Class ChplAPIPage definition.
+ */
+public final class ChplAPIPage {
```

Comment:
```suggestion
public final class ChplAPIPage extends BasePage {
```

This class needs to extend the BasePage, then any calling step can do `ChplApiPage.mainContent` and get that element